.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,19 +2,34 @@
 name: Feature request
 about: Suggest an idea for this project
 title: ''
-labels: New Feature
+labels: language feature request
 assignees: ''
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+<!--
+This repository tracks issues and requests related to the implementation of the
+Swift compiler, standard library, runtime, and tools that provide IDE support
+for Swift (e.g. code completion). If your feature request relates to the
+implementation of a proprietary (closed-source) Apple framework such as UIKit,
+SwiftUI, Combine, etc., please report it to https://feedbackassistant.apple.com
+instead.
+-->
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+**Motivation**
+<!-- Describe the problem that your feature seeks to address. -->
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+**Solution**
+<!--
+Describe your solution to the problem. Provide examples and describe how
+they work.
+-->
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+**Alternatives considered**
+<!--
+Describe any alternative approaches or features that you have considered in
+addressing the problem, and why you chose this approach instead.
+-->
+
+<!-- Add any other context about your feature as appropriate. -->
+<!-- **Additional context** -->
```

Comment:
Should we have a Swift Forums reference here as well? Something saying that people could report the feature request here, but will be nice if they open a discussion in the forums(if that is a language feature). Or if they already have add link here for reference. Do you think this makes sense?

Reply:
Yes, good point.